pallets/issue/src/tests.rs

```diff
@@ -447,14 +447,24 @@ fn test_set_issue_period_only_root() {
 #[test]
 fn test_request_issue_fails_exceed_limit_volume_for_issue_request() {
 	run_test(|| {
-		let volume_limit = 1u128;
+		let volume_currency = DEFAULT_COLLATERAL_CURRENCY;
+		let volume_limit = 1_000u128;
 		crate::Pallet::<Test>::_rate_limit_update(
 			std::option::Option::<u128>::Some(volume_limit),
-			DEFAULT_COLLATERAL_CURRENCY,
+			volume_currency,
 			7200u64,
 		);
 
-		let issue_amount = volume_limit + 1;
+		let issue_asset = VAULT.wrapped_currency();
+		// First, convert the volume limit to the issue asset
+		let volume_limit_denoted_in_wrapped_asset =
+			Oracle::convert(&Amount::new(volume_limit, volume_currency), issue_asset)
+				.expect("Price conversion should work");
+
+		// We set the issue amount as the volume limit + 100 (we have to use at least 100 because
+		// the issue_amount might get rounded down during the price conversion between assets with
+		// decimals differing by 2)
+		let issue_amount = volume_limit_denoted_in_wrapped_asset.amount() + 100;
```

Comment:
I think so. If the wrapped vs collateral assets that we use in the test cases differ by more than 2 decimals we'd probably encounter issues again. I guess the perfect solution would be something similar to how I calculate the tolerance [here](https://github.com/pendulum-chain/spacewalk/blob/e427fbd43f55766dfecd95fbcbb44225b161a2bc/pallets/vault-registry/src/tests.rs#L1103-L1108). Doing something similar everywhere would reduce readability but increase the robustness in case we change the decimals in the future. Since this is quite unlikely to happen, I'm a little hesitant but what do you think @gianfra-t?

Reply:
Alright ! I don't think it is necessary either to add the tolerance, just wanted to confirm my understanding. As long as we are aware it is fine.